scripts/basic-test.sh

```diff
@@ -18,7 +18,7 @@ for k, v in sorted(dict(os.environ).items()):
 pip install .
 
 
-PYTEST="python -m pytest"
+PYTEST="python -m pytest -n2"
```

Comment:
I'm not sure defining this here is actually a space saving vs explicitly opting into it in the places needed. It's resulted in an awful lot of `-n0` arguments in tests/pytest that seem mostly to just be noise!

One option would be to do something like:

```
PYTEST_NOXDIST="python -m pytest"
PYTEST="$PYTEST_NOXDIST -n2"
```

And then use "PYTEST_NOXDIST" in the places that don't need it if those really are the minority (which I guess they are).

Reply:
hey @DRMacIver, ugh, I'm sorry about. The `-n0`s are no longer necessary since I followed your suggestion in https://github.com/HypothesisWorks/hypothesis-python/pull/1123#issuecomment-366906248.

I thought I had removed them once they could be removed, but hadn't. 😦

Reply:
(they're gone now)